app/assets/javascripts/discourse/app/controllers/user-private-messages-group.js

```diff
@@ -11,15 +11,17 @@ export default class extends Controller {
 
   @computed(
     "pmTopicTrackingState.newIncoming.[]",
-    "pmTopicTrackingState.statesModificationCounter"
+    "pmTopicTrackingState.statesModificationCounter",
+    "pmTopicTrackingState.isTracking"
```

Comment:
Do we know the rationale of why isTracking is here in the first place?

Reply:
Is isn't here in the first place because we were blocking the rendering of the route until the model resolves. Therefore, there is no need to check that `isTracking` has been updated previously.